.github/workflows/countline.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+#get current working dir
```

Comment:
1. Put code in a [main function](https://realpython.com/python-main-function/)
2. Use [argparse](https://docs.python.org/3/library/argparse.html) for specifying the root directory to start at with an optional default. Argparse is for specifying CLI options for the script.
3. Use [argparse](https://docs.python.org/3/library/argparse.html)  to specify the number of lines of code, with a 300 default. Argparse is for specifying CLI options for the script.

Reply:
For the directory location, I have set the required value to False but haven't use the deafult value in add argument itself instead I used `os.getcwd()`

```
parser = argparse.ArgumentParser()
parser.add_argument('--line', type=int, required=False, default=300, help='an integer for number of lines of code')
parser.add_argument('--dir', type=str, required=False, help='Script Location')
args = parser.parse_args()

path = args.dir
if path == None:
    path = os.getcwd()
```